persist.cabal

```diff
@@ -25,13 +25,20 @@ source-repository head
   type:     git
   location: git://github.com/minad/persist
 
+flag force-alignment
+        manual: True
+        default: False
+
 library
         default-language:       Haskell2010
 
-        build-depends:          base >= 4.4 && < 5, containers,
+        build-depends:          base >= 4.7 && < 5, containers,
                                 bytestring >= 0.10.4 && < 1,
                                 text >= 1.2 && < 1.3
 
+        if flag(force-alignment) || arch(PPC) || arch(PPC64) || arch(Mips) || arch(Sparc) || arch(Arm)
+                cpp-options: -DALIGNED_MEMORY
+
```

Comment:
I would prefer whitelisting of unaligned access. `flag force-unaligned` and then a whitelist.

Reply:
Will do. Probably tomorrow.